docs/ota-client-guide/modules/ROOT/pages/account-setup.adoc

```diff
@@ -8,24 +8,25 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 endif::[]
 
 
-If you do not want to mix up test and production software, you may use separate user logins or create additional environments.
+An environment is your working space in your https://connect.ota.here.com[OTA Connect account], where you can create and manage software update projects.
 
-== Separate user logins
+If you do not want to mix up test and production software xref:evaluation-to-prod.adoc#_integrate_ota_connect[during integration with OTA Connect], you may create additional environments. You can xref:ota-web::create-environment.adoc[create] up to 10 additional environments in one https://connect.ota.here.com[OTA Connect account].
 
-To manage the different stages of production workflow, you can create separate user logins:
+For example, you may need to have different environments for development, QA, and production.
+
+IMPORTANT: The limit of 10 additional environments applies even if you leave the environments that you have created. There is currently no way to delete environments.
+
+== Additional environments
```

Comment:
Since you no longer provide two possible ways in which users may get additional environments, there is no need to divide this topic into subsections.  This part mainly describes why it is convenient to use additional accounts and may be combined with what's written above.